src/reuse/modules/util/formatter.ts

```diff
@@ -156,64 +156,44 @@ export class Formatter {
     let mm: number | string = date.getMonth() + 1;
     const month = date.toLocaleString(locale, { month: "short" });
     const yyyy = date.getFullYear();
-
-    if (sec < 10) {
-      sec = `0${sec}`;
-    }
-
-    if (min < 10) {
-      min = `0${min}`;
-    }
-
-    if (hour < 10) {
-      hour = `0${hour}`;
-    }
-
-    if (dd < 10 && format !== DateFormats.MONTH_DAY_YEAR_COMMA_SHORT) {
-      dd = `0${dd}`;
+    let yy: number | string = yyyy % 100;
+
+    sec = sec.toString().padStart(2, "0");
+    min = min.toString().padStart(2, "0");
+    hour = hour.toString().padStart(2, "0");
+    dd = (dd < 10 && format !== DateFormats.MONTH_DAY_YEAR_COMMA_SHORT) ? dd.toString().padStart(2, "0") : dd;  
+    mm = mm.toString().padStart(2, "0");
+    yy = yy.toString().padStart(2, "0");
+
+
+    const formatMappings = {
```

Comment:
I like this constant, but I would keep it outside of the class, just to make the function a bit shorter and easier to read?

Reply:
actually i didnt changed that at all, think it is showing as new code as it got slightly moved?

Reply:
can we move it out nevertheless as Vlad suggested?

Reply:
as said we will do that in another PR